### ISSUE TITLE
Paste inserts duplicate content when using a textarea in MacOS App

### ISSUE THREAD

**Issue by syron** [home-assistant/iOS#2068]
**Device model, version and app version**

Model Name: MacBook Pro (13 tum, M1, 2020)
macOS Version: 12.1
App Version: 2021.322
Safari Version: 15.2 (17612.3.6.1.6)

Home Assistant Core Version
2022.2.2

**Describe the bug**
_The following bug description is only applicable within the Home Assistant App running on MacOS. When using Safari or Chrome, the bug cannot be reproduced._
Whenever pasting a text into a textarea, such as the File Editor or Studio Code Server, the copied text is being pasted two (2) times.

**To Reproduce**
Copy any text. Start he Studio Code Server, go in whatever file you want, paste the copied text. Voilà, the text has been pasted two times.

**Comment by adamfeldman** [comment]
I've been experiencing this for ~months. I regret not reporting the bug sooner.

**Comment by robertoash** [comment]
Any way that this can be prioritized? It makes it necessary to always have a browser window open alongside the MacOS app in order to do edits, making the app basically just a dashboard which is not as useful as it could be.

**Comment by Driekes** [comment]
Indeed, same for me. I need to use the browser for proper text/code adjustments.

**Comment by kdrummer** [comment]
+1

Seeing this on Mac app version 2022.358

**Comment by johngianni** [comment]
Copy and Paste is broken in the MacOS app since ages. It will paste text copied from another app, but it will not copy in the clipboard text selected within the various code windows. So so annoying.

**Comment by syron** [comment]
Too bad, actually a reason to go away from homeassistant, but I will check the code myself and see if there is any obvious finding.
